Add new AcadeMeData app

DanWaldman opened the pull request:
- Includes all files generated by startapp command.
- Does not include models - will come in later PR's.
- Added .idea file to git ignore.
- Added app to installed apps in settings.py.

Yarboa commented:
@DanWaldman please review the git challange again
As a reviewer i can not track the history, since you are creating new PR each time 
https://github.com/redhat-beyond/AcadeMe/pull/28, which is closes

Yarboa commented on AcadeMeData/.idea/AcadeMeData.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Please add the .idea to .ignore file

Yarboa commented on AcadeMeData/.idea/inspectionProfiles/profiles_settings.xml:
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
same here for the ignorefile

Omeramsc commented:
As I mentioned before - please add an issue about this feature, and make sure both the issue, commit message and PR have a well-written description which someone who don't know the project can understand. 

- also, make the titles of the PRs shorter and clearer. an example: `Add new AcadeMeData app module`

DanWaldman commented:
An Issue has been opened about this feature.
A new commit adding .idea to ignore file has been uploaded. 

Can the PR be merged now?

Yarboa commented:
> An Issue has been opened about this feature. A new commit adding .idea to ignore file has been uploaded.

Thanks
But still you should apply git rm .idea dir which the files already exist in PR
> 
> Can the PR be merged now?
No see above,
An no comment in this PR

DanWaldman commented:
FINALLY - I've rearranged my commits for them to have a good order and no repetitions.
I added .idea to git ignore and made those filed go away from the depository.
I added all files that were created by startapp command. 
Hopefully models will be coming in next PR's.

DanWaldman commented:
#34

Yarboa commented on AcadeMeData/apps.py:
```diff
@@ -0,0 +1,6 @@
+
+
+class AcademedataConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```
Why do you need this one?

DanWaldman replied:
I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Yarboa replied:
> I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

Yarboa replied:
> I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

DanWaldman replied:
Done
